### PR TITLE
Dictionary Project

### PR DESCRIPTION
**Pull request by Leah2356**
Screen Shot 2024-04-10 at 3 50 09 PM

### CODE REVIEW

File: src/test/java/feiner/dictionary/EnglishDictionaryTest.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+class EnglishDictionaryTest {
+
+    @Test
+    public void getDefinition() throws CsvValidationException, IOException {
+        // given
+        EnglishDictionary dictionary = new EnglishDictionary();
+
+        // when
+        List<String> definitions = dictionary.getDefinition("hello");
```
Comment (andrewoid) [inline src/test/java/feiner/dictionary/EnglishDictionaryTest.java]: Test a word that has multiple definitions

File: src/main/java/feiner/dictionary/GUI.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    }
+
+    private void updateDefinitions(String word) {
+        definitionsArea.setText(""); // Clears prev definitions
+        if (!word.isEmpty()) {
+            List<String> definitions = dictionary.getDefinition(word);
+            if (definitions != null && !definitions.isEmpty()) {
+                for (String d : definitions) {
+                    definitionsArea.append(d);
+                }
```
Comment (andrewoid) [inline src/main/java/feiner/dictionary/GUI.java]: Did this look correct if you try a word with multiple definitions?

**Review by andrewoid — COMMENTED** [review]
Add a readme

File: screenshots/Screen Shot 2024-04-10 at 2.07.59 PM.png
Comment (andrewoid) [inline screenshots/Screen Shot 2024-04-10 at 2.07.59 PM.png:1]: For some reason, github can't render this file.

File: src/main/java/feiner/dictionary/Gui.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    }
+
+    private void updateDefinitions(String word) {
+        definitionsArea.setText(""); // Clears prev definitions
+        if (!word.isEmpty()) {
+            List<String> definitions = dictionary.getDefinition(word);
+            if (definitions != null && !definitions.isEmpty()) {
+                for (String d : definitions) {
+                    definitionsArea.append(d);
+                }
```
Comment (andrewoid) [inline src/main/java/feiner/dictionary/Gui.java]: Does this look right with words that have multiple definitions?

**Comment by Leah2356** [comment]
I added a space between definitions, and it looks right now. Also,
regarding the screenshot, do you want me to post it in the comments because
it's not going through.
Thank you!

File: src/test/java/feiner/dictionary/EnglishDictionaryTest.java (hunk 1 above)
Comment (andrewoid) [reply src/test/java/feiner/dictionary/EnglishDictionaryTest.java]: Please address this comment.

**Comment by Leah2356** [comment]
I addressed it, and I added the screenshot to my comments because it's not
working on readme.

**Review by andrewoid — APPROVED** [review]

**Pull request merged**